swimster-api/swimster-schema.sql

```diff
@@ -32,6 +32,19 @@ CREATE TABLE listings (
     FOREIGN KEY (host_id) REFERENCES users(id) ON DELETE CASCADE
 );
 
+CREATE TABLE reservations (
+    id          SERIAL PRIMARY KEY,
+    user_id     INT NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    listing_id  INT NOT NULL REFERENCES listings(id) ON DELETE CASCADE,
+    date        DATE NOT NULL,
```

Comment:
nit: more specific name: `reservation_date`

Also, this model disallows multi-day reservations by design. Just double-checking that this is intended.